masterdata/returnRequest/schema.json

```diff
@@ -211,6 +210,7 @@
                   "format": "date-time",
                   "required": true
                 },
+                "role": { "type": "string", "required": true },
```

Comment:
You can make it as ENUM too. It adds one more layer of check for us.

Reply:
@spclu6 nice!